### ISSUE TITLE
Qube Manager icons become vertically desynchronized from their rows

### ISSUE THREAD

**Issue by dandondev** [QubesOS/qubes-issues#5114]
**Qubes OS version**
Qubes R4.0.1

**Affected component(s) or functionality**
Qubes VM Manager

**Brief summary**
One of my laptops suddenly got this problem. The "state" is on the wrong VM. It lights up above the VM that it is supposed to be on. 

**To Reproduce**
_Steps to reproduce the behavior:_
_1. Go to Qubes VM Manager
_2. Click on a VM to start.
_3. State Light (green) for ON, (yellow) for LOADING, is on the above listed VM row, confusing. 

**Expected behavior**
State column should light up on the VM row that is turned on.

**Actual behavior**
State Light (green) for ON, (yellow) for LOADING, is on the above listed VM row, confusing. 

**Screenshots**
![snapshot1(1)](https://user-images.githubusercontent.com/52020289/59963785-08961600-94c6-11e9-8dae-145a49622dff.png)

**Comment by andrewdavidwong** [comment]
I've noticed basically the same thing. It also affects the padlock icons. The real problem is not that the icons show up on the wrong rows. Rather, it's that the icons become vertically desynchronized from the rows themselves, becoming offset from their original rows. For me, this results in the icons sometimes showing up *between* rows. In other words, the actual problem is that icons are not staying on the rows, but rather "drifting off." It's some kind of GUI problem.

**Comment by andrewdavidwong** [comment]
By the way, there is a simple workaround, which is to minimize the Qube Manager window, then restore it again (i.e., double click the window button in the panel).

**Comment by dandondev** [comment]
Alright, thanks. It kinda confused me a bit, thinking that a VM wasn't on while it was.

**Comment by brendanhoar** [comment]
I'd suggest leaving this open as the "workaround" isn't really a solution for a  confusing display-related bug.

**Comment by andrewdavidwong** [comment]
> I'd suggest leaving this open as the "workaround" isn't really a solution for a confusing display-related bug.

Yes, this should not be closed. There's definitely a bug to fix here.

**Comment by marmarta** [comment]
This really looks like a Qt bug - I can reproduce it in Qt4.0, but not in Qt5.0 (pull request for this one is waiting for Qubes 4.1); hopefully, Qt version upgrade will fix this bug. I'll leave this issue open and once Qt5.0 is merged into testing we'll see if it really helped.

**Comment by marmarta** [comment]
So, we not only have Qt5.0 in R4.1, but also the beautiful model/view refactor from here https://github.com/QubesOS/qubes-manager/commit/bb1995af2e90aae51f07c518b74d98221c2d5cc1 . I cannot reproduce this issue, so I'll close this and if anyone encounters this problem with current Qube Manager release, please reopen.
